lib/em-hiredis/pubsub_connection.rb

```diff
@@ -21,6 +20,7 @@ def initialize(inactivity_trigger_secs = nil,
         EM::Hiredis.logger.debug("#{@name} - Sending ping")
         send_command('subscribe', PING_CHANNEL)
         send_command('unsubscribe', PING_CHANNEL)
+        # send_command('ping')
```

Comment:
I wanted to implement this as well instead of subscribe / unsubscribe as it's nicer but didn't finish